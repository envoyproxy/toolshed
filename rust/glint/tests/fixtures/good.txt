This is a clean file
With no trailing whitespace
And consistent indentation
Everything is perfect here
Last line has a newline
