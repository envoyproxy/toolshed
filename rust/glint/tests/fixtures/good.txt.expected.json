{
  "files": {},
  "summary": {
    "total_files": 1,
    "files_with_issues": 0,
    "total_issues": 0
  }
}
